pmm-app/docker-compose.yaml

```diff
@@ -0,0 +1,144 @@
+version: '3'
+services:
+  pmm-server:
+    container_name: pmm-server
+    image: perconalab/pmm-server:dev-latest
+    volumes:
+      - "/Users/lunaticus/Documents/Work/Percona/grafana-dashboards/pmm-app:/var/lib/grafana/plugins"
```

Comment:
We should avoid hard-coded configs, I suggest we use an environment variable.
```yml
    volumes:
      - "${PMM_APP_PATH}:/var/lib/grafana/plugins"
```

Reply:
Thanks! this line got here by accident